### PR TITLE
Fix orcpub mutiple rebuilds

### PR DESCRIPTION
**Pull request by datdamnzotz**

### CODE REVIEW

**Comment by MeanderingCode** [comment]
I'm curious what error this addresses?

Whenever I've built, the `ADD` is always run, so the `RUN lein uberjar` is always a new layer.

**Comment by datdamnzotz** [comment]
The behavior I was seeing was doing a 2nd or 3rd build wasn’t updating the pages with changes. The site contact email in the bottom left.  I ran the docker-compose up —build 3 times and there there was no changes.  I don’t think it was browser cache, chrome was in dev mode with cache disabled, and cloud flare was in dev mode as well.  I added the lien clean on my dev box and it worked. Then I pushed to dmv.com and it worked again.  Maybe I did something wrong but it seemed to fix the problem...

**Comment by MeanderingCode** [comment]
This is odd behavior. I would like to do more testing, though in truth it should be negligible time addition to the build process, I just hate unnecessary code. Makes everything harder to understand, such as giving the impression that this is not a new filesystem for those who don't know Docker very well.

**Comment by datdamnzotz** [comment]
Yes it is odd  Perhaps nginx was caching? I'm not sure.  
lein clean then the build and changes showed up.

I've seen the same doing local builds as well.

**Comment by MeanderingCode** [comment]
After further investigation, this turns out to be an nginx caching issue.

Possible reason is the server-wide `/etc/nginx/nginx.conf` file in the image having `sendfile on`, as there are some reports of errant behavior in VM and container environments.
Possible solutions:
* Turn `sendfile off`: undesired from performance perspective
* Turn 'gzip on`: Likely good solution, though implications should be thought through.

Interrim workaround: deploy new `web` container.

Open an issue on this if we want to do some testing on nginx configurations.

File: docker/orcpub/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ RUN lein deps
 
 ADD ./ /orcpub
 RUN printenv &&\
+	lein clean &&\
```
Comment (MeanderingCode) [inline docker/orcpub/Dockerfile:11]: As per conversation, this is not necessary as the filesystem is fresh at this point.
More in review comment.

**Review by MeanderingCode — CHANGES_REQUESTED** [review]
After further investigation, this turns out to be an nginx caching issue.

Possible reason is the server-wide `/etc/nginx/nginx.conf` file in the image having sendfile on, as there are some reports of errant behavior in VM and container environments.
Possible solutions:

Turn `sendfile off`: undesired from performance perspective
Turn `gzip on`: Likely good solution, though implications should be thought through.
Interrim workaround: deploy new web container.

We can open an issue on this if we want to do some testing on nginx configurations, or you can simply do some experimentation with `gzip on`, comment here about any implications, revert your Dockerfile commit and add one with nginx conf changes.

**Comment by datdamnzotz** [comment]
I'll defer to you guys.  It defiantly looked like a caching issue.

**Review by Demonsthere — COMMENTED** [review]
I agree with @MeanderingCode here, as we are working on a clean image, this shouln't be required

**Comment by datdamnzotz** [comment]
after testing it appears to be nginx cache. closed

**Pull request closed without merging**